docs/running-the-show/set-list-window.md

```diff
@@ -20,19 +20,20 @@ page of playbacks for each song in the show.
 ![Set List Window](/docs/images/Set-List-Window.png)
 
 A set list is made up of tracks. When a track is added it automatically
-links to the current page. Pressing the play button next to the track
+links to the current page. Pressing the \[⏵\] button next to the track
```

Comment:
I'm not sure if this is showing correctly for me:
![image](https://user-images.githubusercontent.com/2584408/94375187-c7010b80-0109-11eb-8556-2ce5001daf65.png)
Similiarly it's not a softkey so I'm uncertain if it should use this styling for clarity

Reply:
Hm, it worked for me but maybe this not an official unicode character. However, as I had people asking where the 'play button' was, I'd prefer a visual represantatin, e.g. a **>**

Reply:
Agreed, but I think this would have to be an image which would cause the paragraph to break up. Perhaps if I put a red box round the play button in the large image and refer to that in the text?

Reply:
I've changed it to use a > symbol

Reply:
Great! Alternatively any of these should work too: https://www.w3schools.com/charsets/ref_utf_geometric.asp